conans/cli/commands/remove.py

```diff
@@ -31,7 +31,7 @@ def remove(conan_api: ConanAPIV2, parser, *args):
     args = parser.parse_args(*args)
```

Comment:
I don't see it necessary, but I see more value in a "multiple remote remove" than in a "multiple remote download", said that, I would keep it single remote.

Reply:
Yes, better single remote, both for download and for remove